classes/player/player.gd

```diff
@@ -1,3 +1,4 @@
+class_name PlayerCharacter
```

Comment:
I think just `Player` would be fine? I'm surprised it wasn't already called that, actually. PlayerCharacter is nice too, though. Your call.

Reply:
I think PlayerCharacter is fine considering this is about the character, we can use the term Player for the user's account in the future (if we add such a thing). I generally prefer specific names over generic ones.